### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by ricardo123321**
in this tic-tac-toe project, we used javascript, HTML, and CSS to create a playable game inside the browser making use of functions and DOM

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @ricardo123321 and @JohannRodriguez 
### You have done well so far :clap: However, you have a few issues to fix:-
- [x] Configure the linters are proposed by Microverse, do not change, or use stickler rather github actions. [refer here](https://github.com/microverseinc/linters-config/tree/master/javascript)
- [ ] Code organization is not up to standard, create a directory and put the main files inside like src
- [x] use es6 to create your functions, methods and create objects for game, gameBoard and player
- [ ] Host the project online and share the link on readme file
### Make these changes and submit for another code review. Good luck.
- Happy coding!

By Raymond
Slack @iamraymond

File: main.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      check(xs);
+      turn();
+    } else if (playerTurn.textContent === `Player ${player2} turn` && tableCell.textContent === '') {
+      tableCell.textContent = 'O';
+      os.push(parseInt(cell.dataset.cell, 10));
+      check(os);
+      turn();
+    }
+  });
+});
```
Comment (rOluochKe) [inline main.js:63]: ### use es6 to creating objects, functions as required

File: main.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    const pl = document.getElementById('player-1');
+    pl.setAttribute('placeholder', 'wrong name');
+  } else {
+    player1 = document.getElementById('player-1').value;
+    firstForm.style.display = 'none';
+    secondForm.style.display = 'block';
+    playerTurn.textContent = `Player ${player1} turn`;
+  }
+});
+
```
Comment (rOluochKe) [inline main.js:84]: ### Separate dom manipulation from the app logic

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great progress @team :clap: However, some requirements weren't implemented as requested. Check :arrow_down: 

#### Implementation (required) :warning: 
* [ ] use es6 syntax to create your functions, modules, use const, let, arrow functions
* [ ] It has `game`, `gameBoard`, `player` objects ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/objects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))
* [ ] Keep the application logic separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright)
* [ ] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)
#### General (required) :warning: 
* [ ] Game should be deployed and working properly
### That's it :relieved: When finished, kindly submit a code re-review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Require Change :red_circle: 
### Great Work So Far! :nerd_face: 
### There still some request made that haven't been met.

### Implementation
- [ ] It's messing the game object

### Deploy
- [ ] Your game is not working, after clicking on create player nothing happens.

# Happy Coding :computer: :nerd_face:

File: scr/main.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
+const playerTurn = document.getElementById('turn');
```
Comment (stratospherique) [inline scr/main.js:1]: * [ ] You should not use global code. Your methods are using variables from everywhere, at least you should pass external variables as arguments to those methods :mag: The point is to make your methods, modules, factories testable as a unit.

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great Progress @team :clap: However, some changes need to be applied in order to get approval for this project. Check :arrow_down: 
#### General (required)
* [ ] Project should have a professional structure. Good files and folders naming. It's "src" not "scr", and your css file could be in an "assets" folder inside "src"
#### Behaviour (required) :warning: 
* [ ] Player should be able to restart the game. The restart button is obsolete. 
#### Implementation (required)
- Check the code comments below.

### That's it :relieved: When finished, kindly submit a code re-review request :pray: 

Happy coding :v: 
**_Ahmed Mahfoudh_**

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi, **TEAM**, congratulations on am excellent job done :100: Thanks to your hard work and the help of our TSE's :+1: our project now meets all the minimum requirements :smile: 
Congratulations once again :fireworks: 

Go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](TSE)

**Pull request closed without merging**